packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -361,15 +386,15 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
 
   Widget _buildChildren(BuildContext context, Widget? child) {
     final ExpansionTileThemeData expansionTileTheme = ExpansionTileTheme.of(context);
-    final Color borderSideColor = _borderColor.value ?? Colors.transparent;
+    final ShapeBorder expansionTileBorder = _border.value ?? const Border();
```

Comment:
Should the `Border()` fallback be transparent? Below I see you create Borders with transparent sides as a fallback.

Reply:
Some comments here and there would help folks understand this in the future.

Reply:
> Should the `Border()` fallback be transparent? Below I see you create Borders with transparent sides as a fallback.
Yep, I'll adjust it now.